apps/server/lib/google-drive/src/internal/to-file.ts

```
@@ -0,0 +1,147 @@
+	if (!options?.type) {
+		const type = parts.find(part => typeof part === "object" && "type" in part && part.type);
+		if (typeof type === "string") {
+			options = { ...options, type };
+		}
+	}
+
+	return makeFile(parts, name, options);
+}
+
+async function getBytes(value: BlobLikePart | AsyncIterable<BlobLikePart>): Promise<Array<BlobPart>> {
+	let parts: Array<BlobPart> = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'parts' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)